src/main/java/eod/card/concrete/normal/Sneak.java

```diff
@@ -43,9 +43,7 @@ public void applyEffect(EffectExecutor executor) {
         if(afterEffect) {
             Arrays.stream(assassins)
                 .map(object -> (Assassin) object)
-                .forEach(assassin ->
-                    player.registerListener(new NextDamageDouble(assassin))
-                );
+                .forEach(NextDamageDouble::new);
```

Comment:
New NextDamageDouble objects won't persist, I guess.
Since they are created inside a function and are not assigned to any variable, I guess that they'll be removed as soon as the process leave applyEffect().